### DISCUSSION TITLE
[Wiki] Simplify steps on Configure Script Addition section for one command

### DISCUSSION THREAD

**Discussion by manugildev** [asmvik/yabai#2095] [Ideas]
Replace [wiki/Installing-yabai-(latest-release)#configure-scripting-addition](https://github.com/koekeishiya/yabai/wiki/Installing-yabai-(latest-release)#configure-scripting-addition) steps by just running this command on the terminal.

```shell
sudo touch /private/etc/sudoers.d/yabai && echo "$(whoami) ALL=(root) NOPASSWD: sha256:$(shasum -a 256 $(which yabai) | cut -d ' ' -f 1) $(which yabai) --load-sa" | sudo tee /private/etc/sudoers.d/yabai
```

**Comment by asmvik** [comment]
These steps are very much "overcomplicated" on purpose, because I want users to be aware of what they are doing, instead of just copy-pasting some command.

**Comment by asmvik** [comment]
I have added this information to the wiki as additional information, but I will not remove the previous part.
